Loop/Managers/DeviceDataManager.swift

```diff
@@ -544,8 +544,32 @@ extension DeviceDataManager {
         return pumpManager?.status
     }
 
-    var sensorState: SensorDisplayable? {
-        return cgmManager?.sensorState
+    func glucoseDisplay(for glucose: GlucoseSampleValue?) -> GlucoseDisplayable? {
+        if let glucose = glucose, glucose.wasUserEntered {
+            guard FeatureFlags.cgmManagerCategorizeManualGlucoseRangeEnabled else {
```

Comment:
I don't think categorization would differ based on manual/cgm values, so we should have one categorization routine for each. In this case, we are splitting between Tidepool Loop, which uses the Dexcom CGMManager categorization routine, and DIY Loop, which will use a hardcoded categorization routine for now. But eventually, they will use the same routine in Loop (which might coordinate with the CGMManager in some ways, but the settings and config will be in Loop).

In essence, I'm suggesting that this temporary hardcoded categorization be used for both cgm and fingerstick glucose values if the flag is turned on.

Reply:
Good to know. I'll add this now.